apps/tup-cms/src/taccsite_cms/templates/assets_custom.html

```diff
@@ -35,5 +35,5 @@
 
 <!-- TUP CMS UI Styles -->
 <style>
-    @import url("{% static 'tup_cms/css/site.cms.css' %}");
+    @import url("{% static 'tup_cms/css/tup-cms.css' %}");
```

Comment:
[`site.___.css` naming is deprecated](https://github.com/TACC/Core-CMS/pull/569/files#r1210964078) & `tup-cms.css` is quicker to find in Network tab

![network table distinction](https://github.com/TACC/tup-ui/assets/62723358/a83c7c4c-9762-403a-b3f1-afdd8a94c38a)